awx/ui_next/src/screens/ExecutionEnvironment/ExecutionEnvironmentDetails/ExecutionEnvironmentDetails.jsx

```diff
@@ -46,10 +46,15 @@ function ExecutionEnvironmentDetails({ executionEnvironment, i18n }) {
           value={image}
           dataCy="execution-environment-detail-image"
         />
-        <Detail label={i18n._(t`Description`)} value={description} />
+        <Detail
+          label={i18n._(t`Description`)}
+          value={description}
+          dataCy="execution-environment-detail-description"
```

Comment:
We haven't met as a team to outline this since... July? of 2020, but I think we should sync on our identifier/location/selection logic as a team.

dataCy is the prefered attribute by the cypress team but it's important to the PEAQE team that we implement data-ouia-id's for consistency across the platform.

I feel like we've been waiting for it to become obvious which pattern to follow and we've ended up with several different implementations within the application and test code. Given where we are with the release, we should set up guidelines to follow and allow Dev and QE to add them where necessary